SetupGuide_Plugin_Admin.php

```diff
@@ -159,8 +159,8 @@ public function abbreviate_url( $url ) {
 	 * @see \W3TC\Util_Http::ttfb()
 	 */
 	public function test_pgcache() {
-		if ( wp_verify_nonce( $_POST['_wpnonce'], 'w3tc_wizard' ) ) { // phpcs:ignore WordPress.Security.ValidatedSanitizedInput
-			$nocache = ! empty( $_POST['nocache'] ); // phpcs:ignore WordPress.Security.ValidatedSanitizedInput
+		if ( wp_verify_nonce( Util_Request::get_string( '_wpnonce' ), 'w3tc_wizard' ) ) { // phpcs:ignore WordPress.Security.ValidatedSanitizedInput
```

Comment:
Remove the two phpcs comments to have it validate the changes.

Reply:
Removed 2 PHPCS disable comments.